test/unit/test_autosubmit.py

```diff
@@ -0,0 +1,34 @@
+from textwrap import dedent
+from pathlib import Path
+from autosubmit.autosubmit import Autosubmit
+from test.unit.conftest import autosubmit_config
+from autosubmitconfigparser.config.basicconfig import BasicConfig
+
+import re
+
+def test_as_conf_default_values(tmp_path, autosubmit_config):
+
+    autosubmit_config('a000', {})
```

Comment:
This adds one linter warning due to the missing extra line. The imports are also not sorted, but that's not too important. If you are using an IDE, you should be able to configure it so it warns you or fails the build, etc. (this might help with the big issue of lint errors/warnings you are taking care of --thanks for that!).

If you add an extra space before this function, then that warning will be gone. You can then also add comment, types :grimacing: 

The `tmp_path` fixture is also never used, I think?

```suggestion
import re
from pathlib import Path
from textwrap import dedent
from typing import Callable, Dict

from autosubmit.autosubmit import Autosubmit
from autosubmitconfigparser.config.basicconfig import BasicConfig
from test.unit.conftest import autosubmit_config


def test_as_conf_default_values(autosubmit_config: Callable[[str, Dict], BasicConfig]) -> None:
    """Test some defaults values of the Autosubmit configuration."""

    autosubmit_config('a000', {})
```